docs/kyma/13-02-basic-troubleshooting.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Basic troubleshooting
+type: Troubleshooting
+---
+
+## Console UI password
+
+If you don't set the password for the **admin@kyma.cx** user using the `--password` parameter or you forget the password you set, you can get it from the `admin-user` Secret located in the `kyma-system` Namespace. Run this command:
+
+```
+kubectl get secret admin-user -n kyma-system -o jsonpath="{.data.password}" | base64 --decode
+```
+
+## Installer doesn't respond as expected
+
+If the Installer does not respond as expected, check the installation status using the `is-installed.sh` script with the `--verbose` flag added. Run:
+
+```
+scripts/is-installed.sh --verbose
+```
+
+## Installation successful, component not working
+
+If the installation is successful but a component does not behave in an expected way, see if all deployed Pods are running. Run this command:
+
+```
+kubectl get pods --all-namespaces
```

Comment:
```
helm list --tls
helm status {RELEASE} --tls
```
Would show more information (such as release/component x is not in `Deployed` state

Reply:
It is also worth to execute `helm history {RELEASE} --tls -o yaml` because it shows errors that occured on each revision. Sometimes you have an for example an error where you have a typo in the charts, it will be reported but with next set of retries the origin error will be "covered" with a different error which is  a consequence of the first one like `{RELEASE} already exsits`. Thanks to helm history we can see the root error

Reply:
Added this approach to the old one